open_event/views/views.py

```diff
@@ -278,6 +281,38 @@ def generate_icalender_track(track_id):
 	cal.add_component(track)
 	return cal.to_ical()
 
+@app.route('/api/v1/session/<int:session_id>/reviews', methods=['GET'])
+@auto.doc()
+@cross_origin()
+def get_reviews(session_id):
+    """Returns all session's reviews"""
+    reviews = Review.query.filter_by(session_id=session_id)
+    return ObjectFormatter.get_json("reviews", reviews, request)
+
+@app.route('/api/v1/session/<int:session_id>/reviews', methods=['POST'])
+@auto.doc()
+@cross_origin()
+def post_reviews(session_id):
+    """Post reviews for sessions"""
+    data = json.loads(request.data)
+    if "comment" not in data:
+            data["comment"] = ""
+
+    reviews = DataGetter.get_reviews_by_email(data["email"])
+    print reviews.count()
+
+    if reviews.count() > 0:
+        DataManager.update_review(data, reviews[0], session_id)
```

Comment:
This seems wrong, you are supposed to change the review for a particular session, but you are seemingly updating the first review in your query after filtering them by email.

A session can have many reviews.
A review must belong to user and to a session.
A user can have many reviews for many sessions.